components/internal/Table.tsx

```diff
@@ -82,13 +82,13 @@ export default function Table(props: Props) {
             prepareRow(row);
             return (
               <Tr {...row.getRowProps()} key={row.id}>
-                {row.cells.map(cell => (
+                {row.cells.map((cell, i) => (
                   <Td
                     height="80px"
                     paddingX="0"
                     color="text.secondary"
                     {...cell.getCellProps()}
-                    key={row.id}
```

Comment:
What was the reason for this change? I feel like the key might get duplicated because we're calling this `map` function on every row, which means the `i` values will repeat.

Reply:
row.id is already used as a key above on line 84. It was causing a bunch of errors in the console. 

Also i read that keys only need to be unique with their siblings and not globably unique so this should be okay? https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings

Reply:
Sounds good!